app/component/route/route.scss

```diff
@@ -25,7 +25,8 @@ $route-header-favourite-icon-selected-hover-fill-color: $background-color;
 }
 
 .departure-list-header {
-  border-bottom: solid 1px #eef1f3;
+  border-top: 1px solid $light-gray;
+  border-bottom: 1px solid $light-gray;
```

Comment:
Actually 1px solid is better, W3C would say :)
https://www.w3.org/community/webed/wiki/CSS_shorthand_reference